doc/GeneratorMigration/Library_Inventory.ps1

```diff
@@ -0,0 +1,394 @@
+# Azure SDK for .NET Libraries Inventory Generator
```

Comment:
since we have this PowerShell script, shall we remove the old https://github.com/Azure/azure-sdk-for-net/blob/main/doc/GeneratorMigration/Library_Inventory.py in this PR?